### PR TITLE
[webview_flutter] Fix issues 25767/34248

### PR DESCRIPTION
**Pull request by stevemoreau**
## Description

*Fix crash when a webview contains a select and user tries to change the value*

## Related Issues

*https://github.com/flutter/flutter/issues/25767 and https://github.com/flutter/flutter/issues/34248*

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Comment by stevemoreau** [comment]
@googlebot I fixed it.

File: packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java [hunk 1]
```diff
@@ -42,7 +44,17 @@
...
+    Context activityContext = context;
+    Context appContext = context.getApplicationContext();
+    if (appContext instanceof FlutterApplication) {
+      Activity currentActivity = ((FlutterApplication) appContext).getCurrentActivity();
+      if (currentActivity != null) {
+        activityContext = currentActivity;
+      }
+    }
+
+    webView = new InputAwareWebView(activityContext, containerView);
```
Comment (amirh) [inline packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java]: We must instantiate the WebView with the presentation context, the activity context isn't tuned for the VD size, doesn't use our custom window manager, and won't use our IMM wrapper.

Comment (stevemoreau) [reply packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java]: Thanks @amirh , another person have made another PR: https://github.com/flutter/plugins/pull/2081/commits/8fc8eb62edf39c1f8db8282c1a1d9661c5f734e2

If I understand your comment correctly, this approach is not right as well. Do you confirm ?

**Comment by stevemoreau** [comment]
@googlebot I fixed it.

**Comment by stevemoreau** [comment]
@googlebot I consent.

**Comment by stevemoreau** [comment]
@googlebot I consent.

File: packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java (hunk 1 above)
Comment (stevemoreau) [reply packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java]: Yeap, I confirm uldaman's PR quoted in previous comment crashes at least on
- API 23 real device
- API 24 emulator

**Comment by amirh** [comment]
Closing per comment on the vd context above.

**Pull request closed without merging**